python/ray/actor.py

```diff
@@ -671,6 +671,10 @@ def __del__(self):
             finally:
                 self.__ray_terminate__._actor_hard_ref = None
 
+    def force_kill(self):
```

Comment:
doc?

Reply:
btw this is async right? any way to confirm actor death?

Reply:
Yes it's async. You can wait on any outstanding actor tasks.